src/app/components/address-displayer/form-address-displayer.tsx

```diff
@@ -1,12 +1,12 @@
 import { SharedComponentsSelectors } from '@tests/selectors/shared-component.selectors';
-import { Box, BoxProps } from 'leather-styles/jsx';
+import { Box } from 'leather-styles/jsx';
 
 import { AddressDisplayer } from '@app/ui/components/address-displayer/address-displayer';
 
-interface FormAddressDisplayerProps extends BoxProps {
+interface FormAddressDisplayerProps {
   address: string;
 }
-export function FormAddressDisplayer({ address, ...rest }: FormAddressDisplayerProps) {
+export function FormAddressDisplayer({ address }: FormAddressDisplayerProps) {
```

Comment:
I refactored this on my travels as we don't need the `...rest` and in case `address` here is undefined and the cause of the runtime error